HW4/kondnataliia/HW4_1.py

```diff
@@ -0,0 +1,15 @@
+number = int(input("Enter your number: "))
+factorial = 1
+if number > 0:
+    for element in range(1, number+1):
+        factorial = factorial * element
+    print(f"The factorial of six is {factorial}.")
```

Comment:
incorrect message, why outputs a factorial of six?

Reply:
Because I'm inattentive. I tested my code on the number six and, apparently, automatically inserted six into the message(
Maybe I can print like this?

```suggestion
    print(f"The factorial of the number you entered is {factorial}.")
```

Reply:
Or like this?

```suggestion
    print(f"The factorial of {number} is {factorial}.")
```